docs/content/developers/design-keuzes.md

```diff
@@ -190,3 +190,17 @@ RGBZ. BESLUIT krijgt een optionele relatie naar ZAAK.
 
 De verwachting is dat er later vergelijkbare designkeuzes gemaakt worden voor
 andere resources, zoals bijvoorbeeld klantcontacten.
+
+## Zaak afsluiten
+
+In de huidige ZDS 1.x standaard is er nog [geen eenduidig besluit genomen over hoe een zaak wordt afgesloten](https://discussie.kinggemeenten.nl/discussie/gemma/koppelvlak-zs-dms/afsluiten-van-een-zaak). 
+
+Wij hebben gekozen voor grofweg de volgende aanpak:
+* Objecttype STATUSTYPE krijgt een nieuw attribuut: `isEindStatus` (ja/nee). Er mag maximaal 1 eindstatus zijn per ZAAKTYPE.
+* Het attribuut `Einddatum` op ZAAK wordt alleen-lezen.
+* Bij het toekennen van een STATUSTYPE met `isEindStatus=ja` aan ZAAK wordt de `ZAAK.Einddatum` automatisch ingevuld met de waarde uit `STATUS.datumStatusGezet`.
+
```

Comment:
Dit is eigenlijk overbodig. Want als er maar 1 eindstatus is kan de einddatum van de zaak afgeleid worden uit de datumStatusGezet van de (enige) eindstatus.

Reply:
@michielverhoef heeft gelijk dat het redundant is, maar vanuit API perspectief is het wel erg fijn als dit werkt en overzichtelijk. Helaas ontbreekt `ZaakType` waardoor we niet binnen het component weten of het een eind status betreft. Dit zal dan opgehaald moeten worden (kan direct tijdens validatie stap). Deze informatie zorgt er dan voor dat Einddatum gezet wordt.

@sergei-maertens Ik pleit er voor om dit ook gewoon te fixen in RGBZ: datumStatusGezet moet DATUMTIJD worden. Hier moeten we echt niet moeilijk over doen, dat zat gewoon fout.

Reply:
Inderdaad, ik was in de war door de naam van het attribuut denk ik.